src/PrestaShopBundle/Utils/ImmutableFloat.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace PrestaShopBundle\Utils;
+
+/**
+ * Immutable value-object that contains a float value.
+ */
+class ImmutableFloat
```

Comment:
> a "real VO" has to be in a VO namespace? O_o Evans knows this?

LOL I meant that as a "proper" VO having domain sense (it's a figure of speech). I don't think a Float VO makes sense from a domain standpoint because it's as generic as it gets.

> The conflict between float and Float is not IMHO a reason to add Immutable

Why not?

> moreover, core classes such as \DateTime have Immutable as suffix

So if it was FloatImmutable it would be OK?

> you are putting something resembling a Type, a VO, a Class into the "Utils" namespace

I agree that Utils may not be the best place to put it. What do you suggest?